### PR TITLE
test: add mysql-schema sample (devservices)

### PR DESCRIPTION
**Pull request by scrocquesel**
Alternative to https://github.com/quarkiverse/quarkus-operator-sdk/pull/201 using Quarkus DevServices (test containers) instead of mocking mysql service class.

### CODE REVIEW

**Comment by metacosm** [comment]
Do you think we should have both versions of the test?

**Comment by scrocquesel** [comment]
> Do you think we should have both versions of the test?

No. it works the same and it promotes usage of devservices. It's just a bit slower to run.

**Comment by metacosm** [comment]
So, to be clear: this is the PR you want merged and not the other one?

**Comment by scrocquesel** [comment]
i'm rebasing

**Comment by scrocquesel** [comment]
This sample has proved useful. See #225

File: samples/mysql-schema/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  you, in particular the CRDs associated with the custom resources we will be dealing with. These
+  CRDs are generated in `target/kubernetes`.
+- Deploy the CRDs to your cluster (requires cluster admin privileges, while this shouldn't be an
+  issue for most "testing" clusters such as `minikube` or `kind`, you might need to log in to your
+  OpenShift clusters with an admin account):
+  ```sh
+  kubectl apply -f target/kubernetes/mysqlschemas.mysql.sample.javaoperatorsdk-v1.yml
+  ```           
+- If you look at the application.properties, you will notice this is where the access to the MySQL server is configured.
+  Obviously, such a server should be running. For eg. `docker run -d --name some-mysql -e MYSQL_ROOT_PASSWORD=password -p 3306:3306 mysql:5`
```
Comment (scrocquesel) [inline samples/mysql-schema/README.md:41]:
```suggestion
  In dev mode, such a server is provided by [Quarkus DevService](https://quarkus.io/guides/dev-services).
```

**Pull request merged**